pandas/core/frame.py

```diff
@@ -893,10 +897,10 @@ def items(self) -> Iterable[Tuple[Optional[Hashable], Series]]:
                 yield k, self._ixs(i, axis=1)
 
     @Appender(_shared_docs["items"])
-    def iteritems(self):
+    def iteritems(self) -> Iterable[Tuple[Optional[Hashable], Series]]:
```

Comment:
Can you create an alias named `Label` for `Optional[Hashable]` in pandas._typing and use that instead? I think should help with readability

Reply:
I've had that thought myself, but I'm starting to have second doubts from a user perspective. E.e. if if an experienced python user sees ``Optional[Hashable]`` he knows what that is, but will have to dig into the pandas source code to see what ``Label`` means.

Anyway, maybe that can go in a seperate PR?

Reply:
Fine as a separate PR. We should aim for abstractions where they help clarify the intent of the annotation and I think `Label` is clearer than `Optional[Hashable]` in that regards